lambda/matching.ml

```diff
@@ -3216,14 +3217,16 @@ and compile_simplified repr partial ctx (m : Simple.clause pattern_matching) =
       (bind_check str v arg lam, total)
   | _ -> assert false
 
-and compile_half_compiled repr partial ctx
+and compile_match_nonempty repr partial ctx
     (m : Typedtree.pattern Non_empty_clause.t pattern_matching) =
   match m with
   | { cases = []; args = [] } -> comp_exit ctx m
   | { args = ((Lvar v as arg), str) :: argl } ->
-      let first_match, rem =
-        split_and_precompile_nonempty v { m with args = (arg, Alias) :: argl }
-      in
+      (* Not sure we need to do anything with arg/v *)
```

Comment:
And I am not sure if we need this comment to blink into existence for one commit.